source/pr_cmds.c

```diff
@@ -1487,330 +1487,286 @@ This is where the magic happens
 =================
 */
 
-int closedset[MAX_WAYPOINTS]; // The set of nodes already evaluated.
-int openset[MAX_WAYPOINTS];//Actual sorted open list
-int opensetRef[MAX_WAYPOINTS];//Reference values of open list
-int opensetLength;//equivalent of javaScript's array[].length;
+
 #define MaxZombies 16
```

Comment:
Is this used? If so, what is this for?

Reply:
It's used to allocate memory to stash a path per-zombie as a list of waypoints.